src/intents.js

```diff
@@ -137,11 +145,24 @@ export function createService (cozy, intentId, serviceWindow) {
       let terminated = false
 
       const terminate = (message) => {
-        if (terminated) throw new Error('Intent service has already been terminated')
```

Comment:
My remark was just for `resizeClient()`: it is the only method which does not terminate the service.